src/harmony/Alert/Alert.module.css

```diff
@@ -0,0 +1,22 @@
+.Alert {
+    display: flex;
+    align-items: center;
+    border-radius: var(--radius-m);
+    padding: var(--gap-sm) var(--gap-sm);
```

Comment:
One var is enough, since they are the same

Reply:
done